packages/cdk-assets/lib/private/handlers/container-images.ts

```diff
@@ -68,6 +52,36 @@ export class ContainerImageAssetHandler implements IAssetHandler {
       file: source.dockerFile,
     });
   }
+
+  protected async getImageUri(ecr: AWS.ECR, destination: DockerImageDestination): Promise<string | null> {
```

Comment:
Write down what the different returned values mean.

Also we generally don't use `null` at all, and use `undefined` instead.